packages/falter-berlin-freifunk-defaults/uci-defaults/freifunk-berlin-freifunk-defaults

```diff
@@ -1,19 +1,19 @@
 #!/bin/sh
 
-uci set freifunk.community.name='berlin'
-uci set freifunk.community.homepage='http://berlin.freifunk.net'
+uci set freifunk.community.name='ukraine'
+uci set freifunk.community.homepage='http://ukraine.freifunk.net'
 
 uci set freifunk.wifi_device=defaults
 uci set freifunk.wifi_device.diversity='1'
 uci set freifunk.wifi_device.disabled='0'
-uci set freifunk.wifi_device.country='DE'
+uci set freifunk.wifi_device.country='UA'
 uci set freifunk.wifi_device.channel='13'
```

Comment:
Maybe we should use something like 11 or even below as wifi-channel. I had experienced, that there are some phones (mainly phones from the US/Canada) that can't use channel 13 even then, if they were switched to a european wifi profile. Using another wifi channel would enable those phones to use the wifi too.

Reply:
Good point.  I guess using channel 10 would be easiest since there is already a default SSID in ad-hoc mode for this channel.